backend/testfiles/execution/cloud/db.dark

```diff
@@ -750,26 +753,42 @@ module FindAll =
                                                                                       "Ross" ]
 
   // inlining user functions
-  let userAnd (user: Person) (human: Bool) : Bool =
-    PACKAGE.Darklang.Stdlib.Bool.and_v0 user human
+  let userAnd (a : Bool) (b: Bool) : Bool =
+    PACKAGE.Darklang.Stdlib.Bool.and_v0 a b
 
   let userStringContains (user: Person) (str: String) : Bool =
     PACKAGE.Darklang.Stdlib.String.contains_v0 user.name str
 
   let userLessThan (user: Person) (height: Int) : Bool =
     Builtin.Int.lessThan_v0 user height
 
-  let nestedUserFn (user: Person) (height: Int) : Bool =
-    userAnd user.human (userLessThan user.height height)
+  let useAdd (a : Int) (b : Int) : Int = a + b
+
 
   (friends (fun p -> userAnd p.human true)) = [ " Chandler "; "Rachel"; "Ross" ]
 
   (friends (fun p -> userStringContains p "Chandler")) = [ " Chandler " ]
 
   (friends (fun p -> userLessThan p.height 65)) = [ "GrumpyCat" ]
 
+  (friends (fun p -> let a = 1 in let b = 9 in (p.height == useAdd 6 4) && (p.age == b) )) = [ "GrumpyCat" ]
```

Comment:
So for this test, I'm trying to see if the `b` parameter of `useAdd` shadows the `b = 9` definition after inlining. However, I don't think this test would trigger it.

Maybe try the following:

```
(friends (fun p -> let a = 1 in let b = 9 in (p.height == useAdd 6 (b - 5)) = [ "GrumpyCat" ]
```

```
(friends (fun p -> let a = 1 in let b = 9 in let c = useAdd 6 4 in (p.height == c) && (p.age == b) )) = [ "GrumpyCat" ]
```